Overmind/settings.py

```diff
@@ -0,0 +1,122 @@
+"""
```

Comment:
Investigate Local settings pattern https://techstream.org/Bits/Local-Settings-in-django